pkg/network/status/status_test.go

```diff
@@ -152,8 +153,8 @@ func TestProbeLifecycle(t *testing.T) {
 
 	// The subsequent calls to IsReady must succeed and return true
 	for i := 0; i < 5; i++ {
-		if ok, err = prober.IsReady(context.Background(), ia); err != nil {
-			t.Fatalf("IsReady failed: %v", err)
+		if ok, err = prober.IsReady(context.Background(), ing); err != nil {
+			t.Fatalf("IsReady returned unexpected error: %v", err)
 		} else if !ok {
```

Comment:
no need for else here. `Fatalf` terminates execution, so just `if`, and simplify the error message.

Reply:
Done